### PR TITLE
Feature/all about reboots

### PR DESCRIPTION
**Pull request by dbuentello**
Still working on this. Hope to be _review ready_ by mid october.

### CODE REVIEW

**Comment by HipsterBrown** [comment]
@dbuentello How's this going? Any way I can help?

File: reboot_article.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## New Features: All About Reboots!
+
+![img1][img1]
+*We've come a long way*
+
+A reboot (or *reset*), on the surface, seems like a trivial tasks, but as the image above illustrates along with the anecdotes below, they present unique challenges of their own.<sup>2</sup>
```
Comment (Frijol) [inline reboot_article.md:6]: typo "seems like a trivial tasks"

**Comment by dbuentello** [comment]
new job, will come back to this at a later date :)

**Comment by Frijol** [comment]
Congrats on the new job!

**Pull request closed without merging**